tests/components/hassio/test_sensor.py

```diff
@@ -28,7 +28,7 @@
 
 
 @pytest.fixture(autouse=True)
-def mock_all(aioclient_mock: AiohttpClientMocker, addon_installed) -> None:
+def mock_all(aioclient_mock: AiohttpClientMocker, addon_installed, store_info) -> None:
```

Comment:
Would be nice to at least add type annotations for the new parameter.

Reply:
Ok I can add it. I usually skipped the type annotations on params added to fixtures that only exist for the purpose of calling other fixtures and aren't actually used in the code. It's unfortunate the `usefixtures` annotation doesn't work for fixtures using other fixtures